src/test/utils/VaultUtils.sol

```diff
@@ -213,6 +271,20 @@ abstract contract BasicVaultTest is UniswapV2Fixture {
         assertEq(getToken0Active(), 0);
     }
 
+    function test_depositAdvanceFuzz(uint128 _amount) public {
+        // note: full uint128 range causes UniswapV2: OVERFLOW error
+        uint256 depositAmnt = toRange(_amount, vault.MIN_LP(), type(uint96).max);
+
+        depositToken0(depositAmnt);
+        depositToken1(depositAmnt);
+
+        advance();
+        assertEq(getToken0Active(), depositAmnt);
+        assertEq(getToken0Reserves(), 0);
+        assertEq(getToken1Active(), depositAmnt);
+        assertEq(getToken1Reserves(), 0);
+    }
```

Comment:
can we fuzz with 2 input variables here?

something like the following where we define a min function

```
function test_depositAdvanceFuzz(uint128 _amount0, uint128 _amount1) public {
        // note: full uint128 range causes UniswapV2: OVERFLOW error
        uint256 depositAmnt0 = toRange(_amount0, vault.MIN_LP(), type(uint96).max);
        uint256 depositAmnt1 = toRange(_amount1, vault.MIN_LP(), type(uint96).max);

        depositToken0(depositAmnt0);
        depositToken1(depositAmnt1);

        advance();

        uint256 depositAmnt = min(depositAmnt0, depositAmnt1);
        assertEq(getToken0Active(), depositAmnt);
        assertEq(getToken0Reserves(), depositAmnt0 - depositAmnt);
        assertEq(getToken1Active(), depositAmnt);
        assertEq(getToken1Reserves(), depositAmnt1 - depositAmnt);
  }
```

Otherwise we're testing a very specific case where the deposit amounts are equivalent

Also, can we increase `type(uint96).max`? This gives us 10^28, but we work with some tokens (e.g. NEAR) that have 24 decimals so this would only give us up to 10_000 tokens, want to provide some more buffer without hitting uniswap's range error. Maybe 1e30

Reply:
good call - this should be good now and using the max possible range given the existing Uniswap pair reserves